src/compo/viirs_l1bnc2ioda.py

```diff
@@ -185,12 +190,15 @@ def _read(self):
                 err.set_auto_scale(False)
 
                 # NASA VIIRS apply secant if option enabled
-                vals[:, ichan] = obs[:].data.ravel() * sec_term if self.apply_secterm else obs[:].data.ravel()
+                vals[:, ichan] = obs[:].data.ravel() * sec_term if not self.stored_refl else obs[:].data.ravel()
```

Comment:
For clarity, can you put this with a more traditional if/else statement with and use 4 lines of code?

Reply:
In general would it be more forward looking to add this secant term in the in ioda file and have either this scaling done in UFO using a yaml key option and/or used as a predictor in VarBC?

Reply:
This would avoid creating multiple datasets with not knowing what it's in there after while.

Reply:
That is a good idea. I feel it can be confused so I added the global attribute to tell whether the data is true reflectance or stored reflectance. How do I add the scaling on observation? I'm working on hofx in UFO/CRTM.

Reply:
Load SZA in UFO, calculate the secant term and apply it.